### PR TITLE
Release 0.20.0-rc.1

### PR DESCRIPTION
**Pull request by JeanMertz**
### Features

- Add `@graphql` directive to generate queries and mutations to an upstream GraphQL API.
- Add `JWKS` provider.
- Initial support for TypeScript configuration.

### CODE REVIEW

File: cli/changelog/0.20.0-rc.1.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### Features
+
+- Add `@graphql` directive to genrate queries and mutations to an upstream GraphQL API.
+- Add `JWKS` provider.
```
Comment (obmarg) [inline cli/changelog/0.20.0-rc.1.md:4]:
```suggestion
- Add `JWKS` auth provider.
```

File: cli/changelog/0.20.0-rc.1.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+### Features
+
+- Add `@graphql` directive to genrate queries and mutations to an upstream GraphQL API.
```
Comment (obmarg) [inline cli/changelog/0.20.0-rc.1.md:3]:
```suggestion
- Add `@graphql` directive to generate queries and mutations to an upstream GraphQL API.
```

File: cli/changelog/0.20.0-rc.1.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+- Add `JWKS` provider.
+- Initial support for TypeScript configuration.
```
Comment (yoav-lavi) [inline cli/changelog/0.20.0-rc.1.md:5]:
```suggestion
### Features

- Adds the `@graphql` directive to support generating queries and mutations from an upstream GraphQL API
- Adds a `JWKS` provider
- Adds initial support for TypeScript configurations
```

File: cli/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [0.20.0-rc.1] - 2023-05-17
```
Comment (yoav-lavi) [inline cli/CHANGELOG.md:3]: No need to add RC changelogs in the main one

**Review by yoav-lavi — CHANGES_REQUESTED** [review]
See comments

File: cli/CHANGELOG.md (hunk 4 above)
Comment (yoav-lavi) [reply cli/CHANGELOG.md:3]: Updated the docs to reflect this

**Pull request merged**